### ISSUE TITLE
Build failure: picard

### ISSUE THREAD

**Issue by jansol** [NixOS/nixpkgs#325946]
### Steps To Reproduce

1. build picard

### Build log

```
error: builder for '/nix/store/mnmph21c8dkl6fwljd66wwxxwhhaw4ja-picard-2.11.drv' failed with exit code 139;
       last 10 log lines:
       > Running phase: pythonCatchConflictsPhase
       > Running phase: pythonRemoveBinBytecodePhase
       > Running phase: pythonImportsCheckPhase
       > Executing pythonImportsCheckPhase
       > Running phase: setuptoolsCheckPhase
       > Executing setuptoolsCheckPhase
       > evaling implicit 'preCheck' string hook
       > running test
       > QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/build/runtime-nixbld'
       > /nix/store/jhhxz8gzypmv0pkmzqrcwc7i85jj7qq2-setuptools-check-hook/nix-support/setup-hook: line 4:  1142 Segmentation fault      (core dumped) /nix/store/z7xxy35k7620hs6fn6la5fg2lgklv72l-python3-3.12.4/bin/python3.12 nix_run_setup test
       For full logs, run 'nix log /nix/store/mnmph21c8dkl6fwljd66wwxxwhhaw4ja-picard-2.11.drv'.
```

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.9.7, NixOS, 24.11 (Vicuna), 24.11.20240703.9f4128e`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.4`
 - nixpkgs: `/nix/store/dk2rpyb6ndvfbf19bkb2plcz5y3k8i5v-source`e
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by dschrempf** [comment]
Also happens for `vimiv-qt`.

Probably related: https://github.com/NixOS/nixpkgs/issues/325804

**Comment by doronbehar** [comment]
Hey, `picard` was "fixed" by switching to Python 3.11 at https://github.com/NixOS/nixpkgs/commit/ed05b6c9aab0819e514a4b0d71ccd2f8e9015177 . Hopefully I'll remember to switch back to Python 3.12 when the picard's maintainers will publish a new release and https://github.com/NixOS/nixpkgs/pull/325893 will be merged to `master` by then.
